src/twisted/logger/test/test_logger.py

```diff
@@ -76,6 +76,16 @@ def test_namespaceDefault(self):
         self.assertEqual(log.namespace, __name__)
 
 
+    def test_namespaceOMGItsTooHard(self):
+        """
+        L{CannotDetermineNamespaceError} is raised when unable to determine the
```

Comment:
One should write docstrings in the active voice.  (i.e.: is raised?  raised by whom?)